File: src/main/java/com/plog/domain/post/dto/PostTemplateUpdateReq.java
package com.plog.domain.post.dto;

import jakarta.validation.constraints.NotBlank;

/**
 * postTemplate 을 갱신할 때 사용하는 dto 입니다.
 *
 * @author jack8
 * @since 2026-01-26
 */
public record PostTemplateUpdateReq(
        @NotBlank
        String name,
        @NotBlank
        String title,
        @NotBlank
        String content
) {
}

File: src/main/kotlin/com/plog/domain/post/dto/PostTemplateUpdateReq.kt
package com.plog.domain.post.dto

import jakarta.validation.constraints.NotBlank

/**
 * 템플릿 수정 요청 DTO입니다.
 */
data class PostTemplateUpdateReq(
    @field:NotBlank
    val name: String,
    @field:NotBlank
    val title: String,
    @field:NotBlank
    val content: String,
)
